spec/requests/api/v1/items_request_spec.rb

```diff
@@ -17,7 +17,7 @@
       expect(item[:attributes][:name]).to be_a(String)
       expect(item[:attributes][:description]).to be_a(String)
       expect(item[:attributes][:count]).to be_an(Integer)
-      expect(item[:attributes][:category]).to be_an(Integer)
+      expect(item[:attributes][:category]).to be_an(String)
```

Comment:
I thought we're dealing with integers on the BE for enums, why are we expecting a string here?